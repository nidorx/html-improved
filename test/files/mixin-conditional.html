<mixin name="article" params="title, authorName">
    <div class="article">
        <div class="article-wrapper">
            <h1>#{title}</h1>
            <span>#{authorName}</span>
            <if cond="$content">
                !{$content}
            </if>
            <if cond="!$content">
                <p>No content!</p>
            </if>
        </div>
    </div>
</mixin>

<article title="Hello world" author-name="John Doe" />
<article title="Hello world" author-name="John Doe">
    <p>This is my</p>
    <p>Amazing article</p>
</article>