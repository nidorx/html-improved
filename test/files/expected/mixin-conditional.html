<div class="article">
    <div class="article-wrapper">
        <h1>Hello world</h1>
        <span>John Doe</span>
        <p>No content!</p>
    </div>
</div>
<div class="article">
    <div class="article-wrapper">
        <h1>Hello world</h1>
        <span>John Doe</span>
        <p>This is my</p>
        <p>Amazing article</p>
    </div>
</div>